### PR TITLE
fibbonaci_function

### PR DESCRIPTION
**Pull request by NeoBauer**

### CODE REVIEW

File: module1/homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,32 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+
+        if(sequence==0) return 0;
+
+        if(sequence==1) return 1;
```
Comment (mariomoore) [inline module1/homework/fibonacci/fibonacci.hpp:7]: Ten kod można zamienić na:
`if (sequence <= 1) return sequence;`

Comment (NeoBauer) [reply module1/homework/fibonacci/fibonacci.hpp:7]: Tutaj o tyle widzę problem, że nie podajemy co powinien wyświetlić, jak chcemy uzyskać pierwszy (czyli zerowy) element ciągu, którym jest 0. Z tego kojarzę, ciąg również liczymy od zerowego elementu.

**Comment by NeoBauer** [comment]
> Please fix code formatting. Try to keep proper indenation (4 spaces is good habit), add space before and after operators. That will make your code more readable :)



> Please fix code formatting. Try to keep proper indenation (4 spaces is good habit), add space before and after operators. That will make your code more readable :)

Right now I am trying to write everything in VIM, so it is a little hard :). But I will try harder :). Spaces are not a problem, but indenation is built-in VIM.

**Comment by kontrybutor** [comment]
@NeoBauer 
You can add these lines in your .bashrc file:
set autoindent
set tabstop=4
set softtabstop=4
set shiftwidth=4
set noexpandtab

Now Vim will care about proper basic formatting. You can look for more awesome vim configuration. There is a lot of Vim stuff in the Internet. If you have any questions, feel free to ask :)

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+	
+   	if(sequence==0) return 0; 
+
+	if(sequence<3&&sequence>0)
+		return 1;
+	
+	return fibonacci_recursive(sequence-2)+fibonacci_recursive(sequence-1);
```
Comment (ziobron) [inline module1/homework/fibonacci/fibonacci.hpp:30]: please apply spaces around operators like < && > == - +

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1430], "points": [2, 6] }

**Pull request closed without merging**